.travis.yml

```diff
@@ -39,12 +39,16 @@ env:
   - PYTHONASYNCIODEBUG=x
   - PYTHONASYNCIODEBUG=
 
+matrix:
+  allow_failures:
+    - python: "nightly"
+
 cache:
   directories:
   - $HOME/.cache/pip
 
 before_cache:
-- rm -f $HOME/.cache/pip/log/debug.log
+  - rm -f $HOME/.cache/pip/log/debug.log
```

Comment:
not related. it's hard to review style changes combined with something actually useful which just gets lost among unrelated changes.

Reply:
ok. I will correct